### DISCUSSION TITLE
Chat window font size adjustable by user, please can this feature be added?

### DISCUSSION THREAD

**Discussion by yeahletters** [jamulussoftware/jamulus#1734] [Ideas]
**Has this feature been discussed and generally agreed?**

**Describe the solution you'd like**
 

**Describe alternatives that have been considered**

**Comment by hoffie** [comment]
Hi,

thanks for this suggestion. I'm moving this to a Github Discussion thread which will be linked from here and will reply there.

**Comment by hoffie** [comment]
Can you explain your use case?

As I understand it, you are referring to the overall text size, right? This should be aligned with outer UI elements in your operating system. Would be increasing the font size there be an option?

**Reply by yeahletters** [reply]
Hi many thanks for your response. I am asking for a friend so I will pass that on to them. Will keep you posted. He is on drums and his display is several feet away so he can't read what's going on in the chat properly while drumming.

**Reply by yeahletters** [reply]
I can only change the overall resolution of my system, not global font size for my system. I'm on Sierra macosx.

**Comment by passing** [comment]
maybe you could solve the issue with OS features like described here https://mcmw.abilitynet.org.uk/macos-10-12-sierra-magnifying-the-screen ?

**Reply by yeahletters** [reply]
Thanks for your response and for your useful suggestion. I have passed on the proposed resolution, but due to the fact that the personal mixer and chat windows are separate, this is not a good workaround.

**Comment by passing** [comment]
I think I can give this a try, but would like to get an approval from the maintainers team first, to not get a rebuff after I have implemented that.

**Reply by gene96817** [reply]
Since I have some musicians with older eyes, giving them the option of having larger font size is very helpful. Younger members find it annoying to have larger font sizes.... yep... they have yet to experience older eyes.

**Reply by yeahletters** [reply]
Thanks a lot, my husband is a developer and he tried but couldn't get the source compiled.

**Comment by atsampson** [comment]
It's a bit non-obvious, but you may be able to adjust the chat window's font size by holding Ctrl and using the scrollwheel on the mouse. (This works on Windows and Linux at least; I haven't tried on MacOS.)

**Reply by yeahletters** [reply]
That would be magnification, if I'm not mistaken. I already suggested this, but that is not going to work for him.

**Reply by yeahletters** [reply]
Just tried testing this, I'm on macos but it doesn't work. Thanks for the suggestion nevertheless!

**Reply by yeahletters** [reply]
I just tried this again. It works for me. It is the cmd+scroll up on the apple mouse. Will suggest this to my drummer friend. Thanks!

**Reply by passing** [reply]
Cool! I'd suggest this to be documented somehow in the [Software Manual](https://jamulus.io/wiki/Software-Manual).
Or maybe still also add a 'Zoom In' and 'Zoom Out' action to the Menu of the Chat Window?

**Comment by yeahletters** [comment]
My husband made a quick custom build. Maybe this feature could be included in future new builds/releases? The issue has now been resolved.
